README.rst

```diff
@@ -138,3 +138,15 @@ The lifecycle configuration file is in :file:`vagrant/clusterhq-vagrant-buildbot
 It was configured by::
 
   gsutil lifecycle set vagrant/clusterhq-vagrant-buildbot.lifecycle.json gs://clusterhq-vagrant-buildbot/
+
+Mac OS X Buildslave
+-------------------
+
+Running tests on OS X requires root priviledges on an OS X machine and for SSH to be configured on this machine.
```

Comment:
The configuration requires root (to configure a new user), but the tests don't run with root or administrator privileges.